spec/spec_helper.rb

```diff
@@ -13,10 +11,10 @@
 
     # Store any environment variables away to be restored later
     @old_env = {}
-    ENV.each_key {|k| @old_env[k] = ENV[k]}
+    ENV.each_key { |k| @old_env[k] = ENV[k] }
 
     if ENV['STRICT_VARIABLES'] == 'yes'
-      Puppet.settings[:strict_variables]=true
+      Puppet.settings[:strict_variables] = true
```

Comment:
is this tweaking of the setup really necessary with modern versions of the gems we use here?